.env.sample

```diff
@@ -168,3 +168,10 @@ SMTP_SECURE=true
 # The default interface language. See translate.getoutline.com for a list of
 # available language codes and their rough percentage translated.
 DEFAULT_LANGUAGE=en_US
+
+# Optionally enable rate limiter at application web server
+ENABLE_RATE_LIMITER=false
+
+# Configure default throttling paramaters for rate limiter
+MAX_ALLOWED_REQUESTS=
+DURATION_WINDOW_IN_SECONDS=
```

Comment:
Remember these are global, `DURATION_WINDOW_IN_SECONDS` is very ambiguous. We should namespace these.

`RATE_LIMITER_ENABLED`
`RATE_LIMITER_DURATION_WINDOW`
`RATE_LIMITER_REQUESTS`

Reply:
Makes sense. Will push this.

Reply:
Fixed!